google-cloud-storage/Gemfile

```diff
@@ -10,6 +10,5 @@ gem "google-cloud-pubsub", path: "../google-cloud-pubsub"
 gem "minitest-reporters", "~> 1.3.5", require: false
 gem "rake"
 
-# Pin minitest to 5.11.x to avoid warnings emitted by 5.12.
-# See https://github.com/googleapis/google-cloud-ruby/issues/4110
-gem "minitest", "~> 5.11.3"
+gem "minitest"
+gem "rubocop-minitest"
```

Comment:
Can we remove this now that changes have been made and look good?

Reply:
Removed.